src/Dapr.Workflow/WorkflowLoggingService.cs

```diff
@@ -0,0 +1,86 @@
+// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.Workflow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Configuration;
+
+    /// <summary>
+    /// Defines runtime options for workflows.
+    /// </summary>
+    internal sealed class WorkflowLoggingService : IHostedService
+    {
+        private readonly ILogger<WorkflowLoggingService> logger;
+        private static readonly HashSet<string> registeredWorkflows = new();
+        private static readonly HashSet<string> registeredActivities = new();
+        private LogLevel logLevel = LogLevel.Information;
+
+        public WorkflowLoggingService(ILogger<WorkflowLoggingService> logger, IConfiguration configuration)
+        {
+            logLevel = string.IsNullOrEmpty(configuration["DAPR_LOG_LEVEL"]) ? LogLevel.Information : ConvertLogLevel(configuration["DAPR_LOG_LEVEL"]);
```

Comment:
With the changes to make log entries `Information`, is any of this needed any longer?